content/desktop/settings/mac.md

```diff
@@ -201,7 +203,8 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 
 > **Note**
 >
-> Docker Desktop also supports the use of [SOCKS5 proxies](../networking.md#socks5-proxy-support).
+> If you are using a PAC file hosted on a web server, make sure to add the MIME type `application/x-ns-proxy-autoconfig` for the `.pac` file extension on the server or website. Without this configuration, the PAC file may not be parsed correctly.
```

Comment:
Looks good @aevesdocker. Since this is applicable to other platforms as well, will be automatically added to the proxy section of other platforms ?